src/lib/y2firewall/clients/installation_finish.rb

```diff
@@ -55,6 +56,13 @@ def modes
       end
 
       def write
+        Yast.import "Mode"
+
+        if Yast::Mode.auto
```

Comment:
Quite simpler than previous PR but.. how it works when there is no firewall section at all and the installation is done through ssh or vnc? I guess it will block the traffic, see https://github.com/yast/yast-firewall/pull/127 and why we needed to add the auto-installation mode back for the finish client even before moving it to the first stage.

Thus, if no section is added at all I would expect the proposal settings to be used instead.

Reply:
hm, good point. didn't try. Will check and adapt.

Reply:
so, in fact it worked (for ssh), however involving product settings was a good point, so I've adapted the path